coalib/coala_modes.py

```diff
@@ -48,6 +48,10 @@ def mode_json(args):
     from coalib.output.printers.LogPrinter import LogPrinter
```

Comment:
commit message:
I think you should point out that you add a `--log-json` flag^^
sth like: `Add --log-json flag` or so